schemas/credentials.json

```diff
@@ -357,23 +357,56 @@
                 "description": "SD-WAN Portal Host password to login into command line",
                 "default": "",
                 "encrypt": true,
+                "sectionEnd": "Portal",
                 "propertyOrder": 360
             },
             "smtp_auth_username": {
                 "type": "string",
                 "title": "SMTP username",
                 "description": "Username for SMTP authentication",
                 "default": "",
+                "sectionBegin": "SMTP",
                 "propertyOrder": 370
             },
             "smtp_auth_password": {
                 "type": "string",
                 "title": "SMTP password",
                 "description": "Password for SMTP authentication",
                 "default": "",
-                "sectionEnd": "Portal",
+                "sectionEnd": "SMTP",
                 "propertyOrder": 380
-           }
+           },
+           "netconf_vm_username": {
+                "type": "string",
+                "title": "Netconf Manager VM username",
```

Comment:
Is NETCONF all caps in general usage? If so, it should be all caps in user-visible titles and descriptions. Not variable names, of course...